templates/song_rater/song_detail.html

```diff
@@ -20,5 +20,12 @@ <h2>Rate The Song</h2>
         {% csrf_token %}
       <button type="submit">Vote</button>
     </form>
+    <ul>
```

Comment:
I would add an h2 or h3 for the list of ratings to explain what we  are looking at